library/src/main/java/com/novoda/storagepathfinder/AndroidCommonDirectories.java

```diff
@@ -12,23 +14,19 @@
  */
 public class AndroidCommonDirectories implements CommonDirectories {
```

Comment:
`CommonDirectories` should probably be changed now, it's referring to the `external` storage directories now.